doc/nrf/drivers/wifi/native/nrf70.rst

```diff
@@ -0,0 +1,115 @@
+
+.. figure:: /images/nrf700x_wifi_driver.svg
+   :alt: nRF Wi-Fi driver block diagram
+   :align: center
+   :figclass: align-center
+
+   nRF Wi-Fi driver architecture overview
+
+Design overview
+***************
+
+The nRF Wi-Fi driver follows an OS agnostic design, and the driver implementation is split into OS agnostic and OS (Zephyr) specific code.
+The OS agnostic code is located in the :file:`${ZEPHYR_BASE}/../modules/hal/nordic/drivers/nrf_wifi/` folder and the Zephyr OS port is located in the :file:`${ZEPHYR_BASE}/drivers/wifi/nrfwifi/` folder.
+
+The driver supports two modes of operation:
+
+Wi-Fi mode
+==========
+In this mode, the driver is designed to be used with the Zephyr networking stack.
+It is implemented as a network interface driver.
+
+The driver supports the following IEEE 802.11 features:
+
+* Wi-Fi 6 (802.11ax) support
+* WPA3™/WPA2™ personal security
+* IEEE 802.11 Power Save modes
+* Scan-only mode
+* IEEE 802.11 :term:`Station mode (STA)`
+* :term:`Software-enabled Access Point (SoftAP or SAP)` mode
+
+The Wi-Fi Direct® mode feature is in the driver code but is not yet supported.
+
+Except for scan-only mode, the driver uses the host access point daemon (hostapd) to implement AP Media Access Control (MAC) Sublayer Management Entity (AP MLME) and wpa_supplicant to implement 802.1X supplicant.
+
+Radio Test mode
+===============
+The nRF Wi-Fi driver supports Radio Test mode, which you can use to test the RF performance of the nRF70 Series device.
+This is a build time option that you can enable using the :kconfig:option:`CONFIG_NRF70_RADIO_TEST` Kconfig option.
+
+For more details about using this driver in Radio Test mode, see :ref:`wifi_radio_test`.
+
+Driver to nRF70 Series device communication
+*******************************************
+
+The driver communicates with the nRF70 Series device using the QSPI/SPI interface.
+The driver uses the QSPI/SPI interface to send commands to the nRF70 Series device, and to transfer data to and from the device.
+The nRF7002 DK uses QSPI, whereas the nRF7002 EK uses SPI.
+
+To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
+
+Configuration
+*************
+
+The nRF Wi-Fi driver has the following configuration options:
+
+Kconfig configuration
+=====================
+
+.. options-from-kconfig::
+   :show-type:
+
+Devicetree Specification configuration
+======================================
+
+The maximum transmit power achieved on a nRF70 Series device-based product depends on the frequency band and operating channel.
+This varies over different :term:`Printed Circuit Board (PCB)` designs.
+
+Multiple calibrations and checks are implemented to ensure consistency across channels and devices.
+However, these values have a dependency on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
+To avoid this problem, you can specify the power ceiling at which the EVM and spectral mask are met for a given PCB design.
```

Comment:
This ensures that the final maximum transmit power won't be above these specified values.